agentstack/generation/tool_generation.py

```diff
@@ -139,3 +145,201 @@ def assert_tool_exists(tool_name: str, tools: dict):
     print(term_color(f'No known agentstack tool: {tool_name}', 'red'))
     sys.exit(1)
 
+
+def _create_tool_attribute(tool_name: str, base_name: str = 'tools') -> ast.Attribute:
+    """Create an AST node for a tool attribute"""
+    return ast.Attribute(
+        value=ast.Name(id=base_name, ctx=ast.Load()),
+        attr=tool_name,
+        ctx=ast.Load()
+    )
+
+def _create_starred_tool(tool_name: str, base_name: str = 'tools') -> ast.Starred:
+    """Create an AST node for a starred tool expression"""
+    return ast.Starred(
+        value=ast.Attribute(
+            value=ast.Name(id=base_name, ctx=ast.Load()),
+            attr=tool_name,
+            ctx=ast.Load()
+        ),
+        ctx=ast.Load()
+    )
+
+
+def _create_tool_attributes(
+        tool_names: List[str],
+        base_name: str = 'tools'
+) -> List[ast.Attribute]:
+    """Create AST nodes for multiple tool attributes"""
+    return [_create_tool_attribute(name, base_name) for name in tool_names]
```

Comment:
I might bring `_create_tool_attribute` into this function if it's not used anywhere else.